### PR TITLE
Adds two new illegal shuttles

### PR DESCRIPTION
**Pull request by Sord213**
# About The Pull Request
Adds two new illegal shuttles that can be bought if the console is emagged or hacked. 

First shuttle is a modified meta shuttle made out of bananium and filled with various clown npcs. I have no idea if these NPCs are balanced or should reasonably see the light of day but I'm letting jesus take the wheel at this point.

Second shuttle is a cluster of small emergency pods. Some are locked for command, security, or medical. All of them make you want to die.

## Why It's Good For The Game
More shuttles are fun. These shuttles are meant to cause you a little pain on the inside if they're bought and you have to experience on them but it's ok sometimes a little pain helps build character.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos (click here to see the pictures,  be advised the bananium shuttle is like a flashbang)</summary>

</details>

## Changelog

:cl:
add: Added two new illegal emergency shuttles. Collect them all!
/:cl:

### CODE REVIEW

**Comment by nednaZ** [comment]
How does the pod swarm work once it launches?
Is the surrounding area still hyperspace, or regular space?

**Comment by Sord213** [comment]
I didn't change any behavior so probably hyperspace. It's intended to be somewhat painful if bought.

**Review by nednaZ — CHANGES_REQUESTED** [review]
Currently unable to load the shuttle due to runtime errors.
"Cannot execute null.canDock()."
My guess is that there is no marker that shows the docking location on the shuttle itself

**Review by nednaZ — CHANGES_REQUESTED** [review]
Still can't get the shuttles to load in properly.
They can be purchased, but the template doesn't seem to exist at all.

File: monkestation/code/datums/shuttles.dm [hunk 1]
```diff
@@ -39,3 +39,23 @@
 	can_be_bought = FALSE
 	admin_notes = "Currently an admin only shuttle. Has working cogscarab shells however they lack the power to start the ark \
 	and cannot teleport unless Reebe is loaded."
+
+/datum/map_template/shuttle/emergency/metabananium
+	prefix = "monkestation/_maps/shuttles/"
```
Comment (nednaZ) [inline monkestation/code/datums/shuttles.dm:44]:
```suggestion
	prefix = "_maps/shuttles/"
```

File: monkestation/code/datums/shuttles.dm [hunk 2]
```diff
@@ -39,3 +39,23 @@
...
+	suffix = "metabananium"
+	name = "Clownified Shuttle"
+	description = "Honk honk honk honk honk honk honk honk honk honk honk honk henk"
+	admin_notes = "Made out of solid bananium, filled with clown mobs. Very evil."
+	credit_cost = 7500
+	can_be_bought = FALSE
+	illegal_shuttle = TRUE
+
+/datum/map_template/shuttle/emergency/pods
+	prefix = "monkestation/_maps/shuttles/"
```
Comment (nednaZ) [inline monkestation/code/datums/shuttles.dm:54]:
```suggestion
	prefix = "_maps/shuttles/"
```

**Review by nednaZ — CHANGES_REQUESTED** [review]
Your issue was that the file path wasn't correct, I got the shuttles loading after making the changes I suggested.

Issue with the pods, they cut off at the point I've highlighted:
ss+(2022-09-15+at+12 18 33)

**Review by nednaZ — CHANGES_REQUESTED** [review]
Last change and it should be good.
Replace the turf passthrough with actual space tiles, so then people can jump between pods in mid flight rather than dropping out into hyperspace

**Review by nednaZ — APPROVED** [review]
Looks good to me.
Will add them as illegal shuttles into the config

**Pull request merged**